x/reward/keeper/grpc_query.go

```diff
@@ -36,14 +36,19 @@ func (q Querier) InflationRate(c context.Context, req *types.InflationRateReques
 
 	baseInflation := q.minter.GetMinter(ctx).Inflation
 	keyManagementInflation := params.KeyMgmtRelativeInflationRate.Mul(baseInflation)
+	validator := req.Validator
 
 	chains := slices.Filter(q.nexus.GetChains(ctx), func(chain nexus.Chain) bool {
 		if !q.nexus.IsChainActivated(ctx, chain) {
 			return false
 		}
 
 		maintainers := q.nexus.GetChainMaintainers(ctx, chain)
-		return len(maintainers) > 0
+		if len(validator) == 0 {
+			return len(maintainers) > 0
+		} else {
+			return slices.Any(maintainers, func(maintainer sdk.ValAddress) bool { return validator.Equals(maintainer) })
```

Comment:
don't actually need to put this inside of an else, can just return since its the end of the function